inc/az_ulib_ipc_api.h

```diff
@@ -56,11 +57,6 @@ typedef struct az_ulib_ipc_tag
   } _internal;
 } az_ulib_ipc;
 
-/**
- * @brief Interface handle.
- */
-typedef void* az_ulib_ipc_interface_handle;
```

Comment:
why was this moved to the descriptor api?

Reply:
Because packages cannot include az_ulib_ipc_api.h, It shall include az_ulib_ipc_interface.h instead, but packages needs this definiction, so, the common point is az_ulib_descriptor_api.h